chia/wallet/nft_wallet/nft_wallet.py

```diff
@@ -782,7 +794,10 @@ async def make_nft1_offer(
         fee: uint64,
         min_coin_amount: Optional[uint64] = None,
         max_coin_amount: Optional[uint64] = None,
+        old: bool = False,
     ) -> Offer:
+        DESIRED_OFFER_MOD = OFFER_MOD_OLD if old else OFFER_MOD
```

Comment:
any thought about wrapping MOD and MOD_HASH into a Class or other structure in order to keep them together?

Reply:
Can you elaborate on the benefit of this? Or is this just a general thought?

Reply:
Mostly a general thought - you can't mix up the two and you don't need to set them individually. I was mostly thinking it avoids a class of possible typos where you use the wrong hash

Reply:
a NamedTuple maybe

Reply:
`namedtuple` is, at least in my mind, primarily a migration tool for moving from regular `tuple` to a full class, presumably a dataclass.  It provides both the old indexing interface that you would be deprecating and the new attribute access you want to move towards.  But I don't think that applies here.

Reply:
This sounds maybe like the `NewType` stuff that Adam is playing around with.

Reply:
this isn't that critical so resolving conversation